notebook/static/notebook/js/completer.js

```diff
@@ -253,12 +253,9 @@ define([
 
             //build the container
             var that = this;
-            this.sel.dblclick(function () {
+            this.sel.click(function () {
                 that.pick();
             });
-            this.sel.focus(function () {
-                that.editor.focus();
-            });
```

Comment:
With all the focus problem we had I would be **extremely** worry of removing something like. 
a few questions: 

On linux/ windows you might need to click on the menu to be able to mouse scroll – does that still work, or can you, as in mac of, scroll even without focusing ? 

With your patch after clicking a completion the notebook is in an inconsistent state. Keybinding are not working correctly, until a reach the mouse and click in a cell, and even though menu can get stuck.


![cantclose](https://cloud.githubusercontent.com/assets/335567/16097218/8a0da628-3301-11e6-9c80-910ca4402b57.gif)

That tend to hint toward the fact that `that.editor.focus()` is here on purpose.